week1/index.js

```diff
@@ -1,25 +1,67 @@
-// To run this file: node index.js
-// Then in your browser: http://localhost:8080
-var http = require('http');
-
-var port = 8080;
-
-var server = http.createServer();
-
-// Start the HTTP server, start listening for requests
-server.listen(port, function(error) {
-  if (error) {
-    console.log(error);
-  } else {
-    console.log('api listening on port', port);
-  }
+/*global console*/
+"use strict";
+//here the global variables
```

Comment:
Section titles are very helpful in long code files, and also helpful in smaller ones. This was a smart move.

You can make them even more obvious, too. I often leave section headers that look like:
```
// ##### GLOBAL VARIABLE DECLARATIONS #####
//     These constants get used throughout the code.
```